kv/cache_snapshot_test.go

```diff
@@ -0,0 +1,153 @@
+// Copyright 2015 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package kv
+
+import (
+	"github.com/juju/errors"
+	. "github.com/pingcap/check"
+)
+
+var _ = Suite(&testCacheSnapshotSuite{})
+
+type testCacheSnapshotSuite struct {
+	k1, k2, k3, k4 []byte
```

Comment:
Keys and values are not always in pairs during the test. These variables are just used for identifying different values.